concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -344,7 +344,7 @@ Hale-1973-1798-342	Animals-45	donkey	n	03a.045	1862	DONKEY
 Hale-1973-1798-343	Animals-46	yak	n	03a.046	2477	YAK
 Hale-1973-1798-344	Animals-47	rhinoceros	n	03a.047	2591	RHINOCEROS
 Hale-1973-1798-345	Animals-48	cat	n	03a.048	1208	CAT
-Hale-1973-1798-346	Animals-49	whiskers	n	03a.049		
+Hale-1973-1798-346	Animals-49	whiskers	n	03a.049	1165	MOUSTACHE
```

Comment:
Since moustache is in category "body" (https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L1166) and I assume "body" means "human body", I would unmap

Reply:
@martino-vic If the mapping is ok from your point of view, you don't need to comment.